ipa-core/tests/common/mod.rs

```diff
@@ -203,7 +203,7 @@ pub fn test_ipa(mode: IpaSecurityModel, https: bool) {
 }
 
 pub fn test_ipa_with_config(mode: IpaSecurityModel, https: bool, config: IpaQueryConfig) {
-    const INPUT_SIZE: usize = 10;
+    const INPUT_SIZE: usize = 100;
```

Comment:
@akoshelev not sure if you wanted to change this SIZE to 100 but 100 shouldn't be too big a number to cause slowdown. Happy to revert if needed